openedx/core/djangoapps/oauth_dispatch/docs/decisions/0016-add-scope-user-id-for-jwt-client-credentials.rst

```diff
@@ -0,0 +1,32 @@
+15. Add scope user_id for JWT token
```

Comment:
* Can you also update the other ADR title?
* Note, I updated the number here too.
* Reminder to update the underline `#####` to be the same length as the title. 
```suggestion
16. Add scope user_id for JWT token and client credentials grant type
```